### PR TITLE
Instructions how does Sharetribe follow Semantic Versioning

### PR DESCRIPTION
**Pull request by rap1ds**
This document adds some instructions how to does Sharetribe follow Semantic Versioning

### CODE REVIEW

File: docs/semantic-versioning.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# How Sharetribe applies Semantic versioning
+
+[Semantic Versioning](http://semver.org/) is a great versioning schema especially for libraries and other software that define a clear *public API*. However, for full-blown application such as Sharetribe, it may not be obvious what is the *public API*.
```
Comment (bladealslayer) [inline docs/semantic-versioning.md]: For *a* full-blown...

Comment (bladealslayer) [inline docs/semantic-versioning.md]: ... what the public API is.

File: docs/semantic-versioning.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+This document is meant to clarify what do we think is the *public API* and how do we change the version number.
```
Comment (bladealslayer) [inline docs/semantic-versioning.md]: ... what we think is the public API and how we update the version number.

File: docs/semantic-versioning.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+*Example:* Require a new database before updating
+
+If you need to do some minor changes to the hosting environment, we will document these steps in [UPGRADE.md](UPGRADE.md). Depending on the size of the change, we may increase the major version number or minor version number.
+
+*Example of a minor change to hosting environment:* Need to set a new configuration value or environment variable value
+
+**User interface**
+
+Major changes to the user interface increate major version number.
```
Comment (bladealslayer) [inline docs/semantic-versioning.md]: s/increate/increment/

**Comment by bladealslayer** [comment]
s/increase/increment/

**Comment by vsaarinen** [comment]
How about examples (or one explanatory sentence) of what patch-level upgrades mean?

**Pull request merged**